File: beginner-projects/beginner/3-strings.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string letters = "abcdefghijklmnopqrstuvwxyz";
    string vowels = "aeiou";
    string input;
    cout << "Enter a string: ";
    cin >> input;
    int vowel = 0;
    int consonant = 0;
    for (int i = 0; i < input.length(); i++)
    {
        if (letters.find(input[i]) != string::npos)
        {
            if (vowels.find(input[i]) != string::npos){
                vowel ++;
            } else {
                consonant ++;
            }
        }
    }
    cout << "In the given string, there are " << vowel << " vowels and " << consonant << " consonants.";
}

File: beginner-projects/beginner/3-strings.py
import string
vowels = 'aeiou'
input = input("Enter a string: ")
vowel = 0
consonant = 0
for char in input:
    if char in string.ascii_letters:
        if char.lower() in vowels:
            vowel += 1
        else:
            consonant += 1
    else:
        break
print(f"In the given string, there are {vowel} vowels and {consonant} consonants.")